File: Packets/src/main/java/me/ImSpooks/core/packets/type/PacketType.java
package me.ImSpooks.core.packets.type;

import lombok.RequiredArgsConstructor;

/**
 * Created by Nick on 01 okt. 2019.
 * Copyright © ImSpooks
 */
@RequiredArgsConstructor
public enum PacketType {

    NETWORK(100),
    DATABASE(200),
    OTHER(300);

    public final int START_ID;
}

File: ClientBridge/src/main/php/client/packets/type/PacketType.php
<?php
/**
 * Created by Nick on 15 okt. 2019.
 * Copyright © ImSpooks
 */

namespace client\packets\type;

class PacketType {
    public const NETWORK = ["NAME" => "NETWORK", "START_ID" => 100];
    public const DATABASE = ["NAME" => "DATABASE", "START_ID" => 200];
    public const OTHER = ["NAME" => "OTHER", "START_ID" => 300];
}